packages/create-nuxt-app/lib/prompts.js

```diff
@@ -112,5 +112,15 @@ module.exports = [
       { name: 'Semantic Pull Requests', value: 'semantic-pull-requests' }
     ],
     default: []
+  },
+  {
+    name: 'vcs',
+    message: 'Version control system:',
+    type: 'list',
+    choices: [
+      { name: 'None', value: 'none' },
+      { name: 'Git', value: 'git' }
+    ],
+    default: 'none'
```

Comment:
Maybe it would be better to keep default as `git` like before? :)

Reply:
@pi0 sure! my reasoning here was that the other fields use `default: 'none'`, so it was an attempt to match those design patterns, but i understand why Git is a good exception 😼

the prompt now uses `git` as default, as well as putting that option at the top of the list

- Use Git as default VCS: https://github.com/nuxt/create-nuxt-app/pull/596/commits/2924cc836eaa0ee59c61a5658797c349e54ccad2